Doc/library/test.rst

```diff
@@ -568,7 +571,7 @@ The :mod:`test.support` module defines the following functions:
 .. function:: disable_gc()
 
    A context manager that disables the garbage collector upon entry and
-   reenables it upon exit.
+   reenables it upon exit, if it was initially enabled.
```

Comment:
(GitHub's not letting me do a suggested change because the line range spans a deleted line.)

I would say something like:

```
   A context manager that disables the garbage collector upon entry. On
   exit, the garbage collector is restored to its prior state.
```

Reply:
Thanks, applying